addons/website_blog/static/src/scss/website_blog.scss

```diff
@@ -244,10 +236,10 @@ $o-wblog-loader-size: 50px;
     #o_wblog_posts_loop {
         .o_record_cover_container {
             box-shadow: inset 0 0 0 1px rgba(white, 0.3);
-            background: rgba(black, 0.1);
```

Comment:
Should this be removed? :thinking:

Reply:
> I also see the background color edition of that cover has simply no effect on that page in master... 

I will put that in a separate commit but this the fix for that bug